### ISSUE TITLE
map on COCO / VOC

### ISSUE THREAD

**Issue by muye5** [experiencor/keras-yolo3#164]
I search over the github, maybe this is the most similar with the darknet implementation.
but I am not sure anybody have reproduced the map on coco or voc reported in the paper.
I tried some kinds of loss, (square loss, bce, softmax, anyway), but could not reproduce the 57.9 mAP50 on coco or even close. any idea is appreciated, I just want to recover the yolo with tf or keras.
